cmd/skaffold/app/cmd/flags.go

```diff
@@ -509,6 +509,14 @@ var flagRegistry = []Flag{
 		DefinedOn:     []string{"debug", "dev", "run"},
 		IsEnum:        true,
 	},
+	{
+		Name:          "build-concurrency",
+		Usage:         "Number of concurrently running builds",
+		Value:         &opts.AssumeYes,
+		DefValue:      1,
```

Comment:
we should only use the value in this flag if it was explicitly set by the user. Only way could be to have the default be `-1` instead, and have a check later if it was overridden by a non-negative integer.

Reply:
Yeah this sounds goo